ci-operator/config/openshift/openshift-tests-private/tools/add_missed_jobs.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
This script should not in this PR, right?

Reply:
um...it was added unexpectedly I will remove it later.